src/pages/login/Login.tsx

```diff
@@ -49,10 +36,13 @@ const Login: FC = () => {
                 </a>
               )}
               {!hasOidc && (
-                <Button hasIcon onClick={openModal} appearance="positive">
-                  <Icon name="security" light />
+                <Link
```

Comment:
I think this should also be a raw `<a href ...` with the other properties, just like the other docs link.